src/analysis/return_value.rs

```diff
@@ -41,6 +44,18 @@ pub fn analyze(env: &Env, func: &library::Function, type_tid: library::TypeId,
         parameter_rust_type(env, func.ret.typ, func.ret.direction, Nullable(false), RefMode::None).is_err()
     };
 
+    let mut bool_return_is_error = !configured_functions.is_empty() && configured_functions.iter().any(|f| f.ret.bool_return_is_error);
+    if bool_return_is_error && func.ret.typ != TypeId::tid_bool() {
+        error!("Ignoring bool_return_is_error configuration for non-bool returning function {}", func.name);
+        bool_return_is_error = false;
+    } else if bool_return_is_error {
+        if env.namespaces.glib_ns_id == namespaces::MAIN {
```

Comment:
I think this could be rewritten as follow:

```rust
imports.add(if env.namespaces.glib_ns_id == namespaces::MAIN { " error" } else { "glib" }, None);
```

However, not sure if this is better...